recenter_tc/version.py

```diff
@@ -0,0 +1,4 @@
+# file generated by setuptools_scm
```

Comment:
@lance-wilson Could you remove this file as well please?  (It is auto-generatred through setuptools_scm)

Reply:
Done.